Makefile

```diff
@@ -11,6 +11,7 @@ KIT ?= bottlerocket-core-kit
 UNAME_ARCH = $(shell uname -m)
 ARCH ?= $(UNAME_ARCH)
 VENDOR ?= bottlerocket
+UPSTREAM_SOURCE_FALLBACK="false"
```

Comment:
Nit: the other assignments in Makefile put spaces before and after the assignment operator. They also do not quote simple strings.